### PR TITLE
Upgrade project

### PR DESCRIPTION
**Pull request by Gauvino**
- Changed from NET 7.0 to 8.0
- Upgraded the version of action 
- Upgraded version of package in csproj

### CODE REVIEW

**Comment by Gauvino** [comment]
@endrl could you look at this, if I need to make some sort of change to it let me know !

**Comment by endrl** [comment]
Hello @Gauvino ,
thank you for the update. Actions can't work as you need to pull my jellyfin fork to build. I am currently waiting for the upstream merge

**Comment by Gauvino** [comment]
> Hello @Gauvino , thank you for the update. Actions can't work as you need to pull my jellyfin fork to build. I am currently waiting for the upstream merge

You are waiting for this to be merged ?
https://github.com/jellyfin/jellyfin/pull/10530

**Comment by endrl** [comment]
yes

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,15 +8,17 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
```
Comment (Gauvino) [inline .github/workflows/build.yml]: @endrl I doubt maybe but does this line seems fair or not ?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -5,11 +5,14 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
```
Comment (Gauvino) [inline .github/workflows/publish.yml]: Same as above

File: .github/workflows/build.yml (hunk 1 above)
Comment (endrl) [reply .github/workflows/build.yml]: Plugins run cross platform and i have no special requirement. The jf plugin template runs just on [ubuntu](https://github.com/jellyfin/jellyfin-meta-plugins/blob/master/.github/workflows/build.yaml).

Comment (Gauvino) [reply .github/workflows/build.yml]: Solved

**Pull request merged**